packages/shared/src/hooks/useToastNotification.ts

```diff
@@ -1,10 +1,14 @@
+import type { ReactNode } from 'react';
 import { useMemo } from 'react';
 import { useQuery, useQueryClient } from '@tanstack/react-query';
 
 type AnyFunction = (() => Promise<unknown>) | (() => unknown);
 
 interface UseToastNotification {
-  displayToast: (message: string, params?: NotifyOptionalProps) => void;
+  displayToast: (
+    message: string | ReactNode,
```

Comment:
`ReactNode` can be a string.
```suggestion
    message: ReactNode,
```